lib/components/user/terms-of-use-pane.js

```diff
@@ -46,11 +48,19 @@ const TermsOfUsePane = ({
         >
           {/* TODO: Implement the link */}
             Optional: I consent to the Trip Planner storing my historical planned trips in order to
-            improve transit services in my area. <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>
+            improve transit services in my area. {termsOfStorageSet && <a href={`/#${TERMS_OF_STORAGE_PATH}`} target='_blank'>More info...</a>}
         </Checkbox>
       </FormGroup>
     </div>
   )
 }
+const mapStateToProps = (state, ownProps) => {
+  return {
+    termsOfStorageSet: state.otp.config.persistence?.terms_of_storage
+  }
+}
+
+const mapDispatchToProps = {
+}
```

Comment:
Remove this var and the second argument of the `connect` function which is optional.

Reply:
Didn't know that thanks! Refactored in 62278865204a45e1afa33483a5742f794c7f5b5e